typescript/test/utils/mock-bridge.ts

```diff
@@ -38,4 +44,24 @@ export class MockBridge implements Bridge {
       resolve(this._difficultyFactor)
     })
   }
+
+  getPendingRedemptions(
```

Comment:
It would be better to name this function `pendingRedemption()`, so that the interface of the `Bridge` is the same as the online contract. However it causes a conflict with the setter `set pendingRedemptions()`.

So I can either keep it as is, or rename the setter to `setPendingRedemption()`, and name the getter `pendingRedemptions()`.

@lukasz-zimnoch What do you think?

Reply:
Yeah, we should name it `pendingRedemptions` to match the Bridge API. Regarding the setter, maybe we can add `requestRedemption` instead? It's also a Bridge's function and it could add a redemption to the mock's `_pendingRedemptions` array.

Reply:
Done in d98dc0c42863be578c7311faa4904cbb46da6eca.